pkgs/by-name/re/redpanda-connect/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+buildGoModule rec {
+  pname = "redpanda-connect";
+  version = "4.47.1";
+
+  src = fetchFromGitHub {
+    owner = "redpanda-data";
+    repo = "connect";
+    tag = "v${version}";
+    hash = "sha256-hyOJf3b6tGB/Wycc7pE70o3pmr+V9hBA+JbDTu+bXws=";
+  };
+
+  proxyVendor = true;
+
+  subPackages = [
+    "cmd/redpanda-connect-community"
+  ];
+
+  vendorHash = "sha256-blBwp4XEuah+rCj0EuOyK4r/DXd5Hv5A7/Z2W5wPZdE=";
+
+  # doCheck = false;
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.Version=${version}"
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  postInstall = ''
+    ln -s $out/bin/redpanda-connect-community $out/bin/redpanda-connect
+  '';
+
+  meta = {
+    description = "Fancy stream processing made operationally mundane";
+    mainProgram = "redpanda-connect";
+    homepage = "https://www.redpanda.com/connect";
+    changelog = "https://github.com/redpanda-data/connect/blob/v${version}/CHANGELOG.md";
+    license = lib.licenses.asl20;
```

```suggestion
    # quoting https://github.com/redpanda-data/connect/blob/main/licenses/README.md
    # Apache-2.0 covers the majority of connectors and functionality.
    # RCL (Redpanda Community License) covers enterprise features.
    license = lib.licenses.asl20; # sub-package "cmd/redpanda-connect-community" is Apache-2.0-licensed
```
So the license is correct if "cmd/redpanda-connect-community" fits in the first category.